packages/react-native-reanimated/Common/cpp/reanimated/Tools/ReanimatedProfilingSection.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+//#define REANIMATED_PROFILING
+
+#ifdef REANIMATED_PROFILING
```

Comment:
Just replace with relevant comment to include it only during profiling. We can also add a GitHub Action to check that no `#include "ReanimatedProfilingSection.h"` is present.

Reply:
With the updated version I want to use the same pattern as RN, i.e. we will have these section in our code permanently, and when `REANIMATED_PROFILING` is not defined, then they will compile to `no-op`.